nuitka/nodes/ImportNodes.py

```diff
@@ -1015,9 +1022,11 @@ def _attemptFollow(self, module_name):
             module_name = resolveModuleName(module_name)
 
             while True:
+                backup_module_name = str(module_name)
```

Comment:
Why would you remember strings ever and not the module name objects. Convert them to strings when time comes.